NNPTPZ1/Mathematics/Complex.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+
+
+
+namespace NNPTPZ1.Mathematics
+{
+    public class Complex
+    {
+        public readonly static Complex Zero = new Complex()
+        {
+            RealPart = 0,
+            ImaginaryPart = 0
+        };
+
+        public double RealPart { get; set; }
+        public float ImaginaryPart { get; set; }
+
+        public Complex Multiply(Complex multiplicand)
+        {
+            // aRe*bRe + aRe*bIm*i + aIm*bRe*i + aIm*bIm*i*i
+            return new Complex()
+            {
+                RealPart = this.RealPart * multiplicand.RealPart - this.ImaginaryPart * multiplicand.ImaginaryPart,
+                ImaginaryPart = (float)(this.RealPart * multiplicand.ImaginaryPart + this.ImaginaryPart * multiplicand.RealPart)
+            };
+        }
+        public double GetAbsoluteValue()
```

Comment:
better to group similiar methods together - getabsvalue + getangleinradians; mathematical methods; ...